crates/oxc_linter/src/snapshots/no_dupe_keys.snap

```diff
@@ -0,0 +1,129 @@
+---
+source: crates/oxc_linter/src/tester.rs
+assertion_line: 53
```

Comment:
Can you update your insta? The latest version doesn't have this anymore.

Reply:
What should I do for this? I have rebase the latest `main` but it still print the lines.

Reply:
`cargo install cargo-insta` or some other cargo update command

Reply:
> `cargo install cargo-insta`

Run this at root but still printing the same result 🤔

Reply:
Should I update `insta` to `1.28.0` ?

```
insta = { version = "1.14.0", features = ["glob"] }
```

Reply:
I'll update it on main.